### ISSUE TITLE
About the issue of pre-trained ckpt unet3d_ckpts.

### ISSUE THREAD

**Issue by CJ416** [Ground-A-Video/Ground-A-Video#6]
So I wonder how you get unet3d_ckpts. As the paper title states, this is a zero-shot method, so we don't need to train. Is unet3d_ckpts composed of GLIGEN groundtokenizer and Unet2D?

**Comment by Ground-A-Video** [comment]
@CJ416 
Yes, Ground-A-Video is a zero-shot method that doesn't require any training. So, the unet3d_ckps compose of SD unet weights and GLIGEN's gated attention weights and groundtokenizer.
